Content.Tests/DMProject/Tests/Procs/procpath_without_proc_element.dm

```diff
@@ -0,0 +1,11 @@
+// RETURN TRUE
+
+// This test confirms we don't OD0404 if a var is set to a procpath that does not contain the "/proc/" element
+
+/atom/movable/proc/foo()
+	return
+
+/datum/var/bar = /atom/movable/foo 
```

Comment:
![image](https://github.com/user-attachments/assets/06fd1f77-b5fd-4cac-8078-c3c99afa345f)
this errors in DM, it's invalid (tries to find a type rather than a proc)

in 515 and later the proper way to call the latest override (lateral or descended) is via `call(nameof(/atom/movable/proc/foo))()` (or `call(nameof(/atom/movable.proc/foo))()`, i don't know why that one is sometimes used? i think it's just a legacy thing)

Reply:
Repeating what I said on discord a while back (to keep any PR observers informed), this does work if it's a proc override. My mistake was not erroring when there's no proc override declared.